src/components/manageMeeting/MeetingInformationForm.tsx

```diff
@@ -53,19 +72,32 @@ const MeetingInformationForm: React.FC<IProps> = ({ meeting, onChange }) => {
         />
       </FormControl>
       <FormControl isRequired fontWeight="normal">
-        <FormLabel sx={labelStyle}>Møtetidspunkt</FormLabel>
+        <FormLabel htmlFor="start-time" sx={labelStyle}>
+          Møtetidspunkt
+        </FormLabel>
         <DatePicker
-          id="published-date"
+          id="start-time"
           selectedDate={meeting.startTime}
           // eslint-disable-next-line
           onChange={(date: any) => onChange({ ...meeting, startTime: date })}
           showPopperArrow={true}
         />
       </FormControl>
+      <Tooltip label="Ved å tillate selvregistrering vil det åpnes en link. Alle som besøker denne linken vil bli lagt til som deltakere på møtet. Det kan endres i ettertid.">
+        <FormControl isRequired width="fit-content" alignSelf="start">
+          <FormLabel sx={labelStyle} htmlFor="self-registration">
+            Tillat selvregistrering
+          </FormLabel>
+          <Switch id="self-registration" onChange={toggleSelfRegistration} isChecked={meeting.allowSelfRegistration} />
+        </FormControl>
+      </Tooltip>
```

Comment:
Display the link when it's generated? This would perhaps make more sense if this section was moved to the participant section. (I did not naturally find where the link was, because when I click on the meeting, the "sub navigation menu" is scolled past automaticly.